docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -240,6 +240,19 @@ Release
       admin/publish-artifacts
       admin/publish-docs --production
 
+#. Check that the artifacts are set up correctly:
+
+   The following command tests that the client packages can be installed on a number of platforms.
+   This helps to identify any problems with the published artifacts that may not be evident in the regular tests (e.g. S3 permissions or packaging problems).
+
+   .. prompt:: bash (flocker-0.1.2)$
+
+      admin/test-artifacts
```

Comment:
I think that's related to https://clusterhq.atlassian.net/browse/FLOC-3044
Maybe that is the error you get when using the Homebrew installed Python?